components/odh-notebook-controller/controllers/notebook_webhook.go

```diff
@@ -324,8 +323,9 @@ func CheckAndMountCACertBundle(ctx context.Context, cli client.Client, notebook
 				},
 			}
 			volumeMounts = append(volumeMounts, customODHCAVolumeMounts...)
+			log.Info("Certificates in 'odh-ca-bundle.crt' are valid and can be mounted")
```

Comment:
This is something that should be on line 316 instead? I mean isn't it duplicit?

Reply:
Hmm, yeah probably is better to keep one Info log for this